themes/powerline-naked/powerline-naked.base.bash

```diff
@@ -25,7 +26,7 @@ function __powerline_left_segment {
 
   LEFT_PROMPT+="$(set_color ${params[1]} -)${pad_before_segment}${params[0]}${normal}"
   LAST_SEGMENT_COLOR=${params[1]}
-  (( SEGMENTS_AT_LEFT += 1 ))
+  ((SEGMENTS_AT_LEFT += 1))
```

Comment:
Sorry my note was regarding reverting ALL changes in this file as they don't relate to the PR.

Reply:
fixed